### ISSUE TITLE
There was an error opening the file

### ISSUE THREAD

**Issue by talesam** [ONLYOFFICE/Docker-DocumentServer#510]
**Do you want to request a *feature* or report a *bug*?**
BUG

**What is the current behavior?**
When I click on create a file and open, the error appears that the file cannot be opened.
I have an installation similarly on another server and it works fine, now I don't know if it's some configuration that's missing., I just configured that server.

**If the current behavior is a bug, please provide steps to reproduce and, if possible, a minimal demonstration of the problem.**

https://user-images.githubusercontent.com/981368/194986151-48d948bf-4520-4eee-9a82-238111cc239d.mp4

**What is the expected behavior?**
Open the file in Onlyoffice.

**Did this work in previous versions of DocumentServer?**
I haven't tested it, it's a fresh install.

**Docker DocumentServer tag:**

**Host Operating System:**
Ubuntu ARM64

## LOG
https://pastebin.com/rKPcJrED

**Comment by ShockwaveNN** [comment]
Seems there is some problems with your dms

Are using nextcloud or owncloud or something else

Could you check that integrated example located at docserverurl/welcome is working fine?

**Comment by talesam** [comment]
I have a server configured like this and it works, I went to configure another one.

Do you refer to this?
https://oo-ruscher.j3d1.xyz
The Onlyoffice container is running there.

**Comment by ShockwaveNN** [comment]
@talesam Yep, you should enabled example as mentioned on this page and check if there will be any errors via example

**Comment by talesam** [comment]
I just tested with the test Onlyoffice and the same error happens.

**Comment by ShockwaveNN** [comment]
Took a look at your video once more

Did you specified JWT key in Nextcloud settings? Since v7.2.0 we got random JWT key by default, info at `/welcome` page

**Comment by ShockwaveNN** [comment]
> Where do I have to configure this?

They are random by default and as I said you can see info about JWT keys on `/welcome` page

**Comment by talesam** [comment]
Let's do it by steps..

1) I'm using version 7.1.1.23, at least when I click on save this is what appears;
2) It didn't work, still with the same exact error. I did the configuration you said, activated it, but it's still giving error.

https://helpcenter.onlyoffice.com/installation/docs-configure-jwt.aspx

**Comment by ShockwaveNN** [comment]
Sorry, I'm out of ideas about what happen, it seems something is wrong on the network layer, maybe some firewall or proxy blocking file download

I do not know what to do next, so I'll leave this issue open, maybe someone else got ideas

**Comment by talesam** [comment]
Network I believe it's no problem, nextcloud and onlyoffice are on the same network, but I'll keep searching, thanks anyway.

If I find the solution I'll post it here.
